envs/mainnet/node.env

```
@@ -1,4 +1,5 @@
 NODE_L1_ETH_RPC=${KROMA_NODE__L1_RPC_ENDPOINT}
+NODE_L1_BEACON=${KROMA_NODE__L1_BEACON_ENDPOINT}
 NODE_L2_ENGINE_RPC=http://kroma-geth:8551
 
```

To enable snap sync behavior, we need to put the flags below.
```suggestion
NODE_SYNCMODE=execution-layer
```

---

Let's make full mode default, and add some additional guides for snap sync.